packages/analytics-js/src/components/pluginsManager/PluginsManager.ts

```diff
@@ -144,29 +144,14 @@ class PluginsManager implements IPluginsManager {
     }
 
     // Consent Management related plugins
-    if (state.consents.activeConsentProviderPluginName.value) {
-      const supportedConsentManagerPlugins: string[] = Object.values(
-        ConsentManagersToPluginNameMap,
-      );
-      pluginsToLoadFromConfig = pluginsToLoadFromConfig.filter(
-        pluginName =>
-          !(
-            pluginName !== state.consents.activeConsentProviderPluginName.value &&
-            supportedConsentManagerPlugins.includes(pluginName)
-          ),
-      );
-    } else {
-      const supportedConsentManagerPlugins: string[] = Object.values(
-        ConsentManagersToPluginNameMap,
-      );
-      pluginsToLoadFromConfig = pluginsToLoadFromConfig.filter(
-        pluginName =>
-          !(
-            pluginName === PluginName.ConsentManager ||
-            supportedConsentManagerPlugins.includes(pluginName)
-          ),
-      );
-    }
+    const supportedConsentManagerPlugins: string[] = Object.values(ConsentManagersToPluginNameMap);
+    pluginsToLoadFromConfig = pluginsToLoadFromConfig.filter(
+      pluginName =>
+        !(
+          pluginName !== state.consents.activeConsentManagerPluginName.value &&
+          supportedConsentManagerPlugins.includes(pluginName)
+        ),
+    );
```

Comment:
Why do we want to compute this if none of the consent managers is enabled by the customer through the load option?

Reply:
By default, all the consent manager plugins are added. If no consent manager is configured, all of them must be filtered.

Reply:
Got it.